test/box-luatest/gh_6260_add_builtin_events_test.lua

```diff
@@ -319,3 +319,86 @@ g.test_box_id = function(cg)
     watcher:unregister()
```

Comment:
Might be better to rename the test file to `builtin_events_test.lua`, without `gh_6260_add` (it is not just ticket 6260 now, and it is really strange to use imperative mood in file names).

Reply:
I don't mind, renamed.